tools/odbc/CMakeLists.txt

```diff
@@ -10,6 +10,9 @@ include_directories(${ODBC_INCLUDE_DIRS} include)
 
 add_library(
   duckdb_odbc SHARED
+  odbc_fetch.cpp # should be first because of foward declaration, i.e., circular
```

Comment:
This seems odd to me. Are you sure this is the case? Unless something funky is happening in the headers the order of `.cpp` files should not matter.

Reply:
I added just as a precaution, but I did a test and the order of .cpp doesn't matter. I will remove that comment.
Thanks for the tip.